endpointmanager/pkg/chplendpointquerier/assurecareparser.go

```diff
@@ -0,0 +1,57 @@
+package chplendpointquerier
+
+import (
+	"io"
+	"strings"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func AssureCareCSVParser(CHPLURL string, fileToWriteTo string) {
+	var lanternEntryList []LanternEntry
+	var endpointEntryList EndpointList
+
+	csvFilePath := "./fhir-base-urls.csv"
+
+	csvReader, file, err := helpers.QueryAndOpenCSV(CHPLURL, csvFilePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	for {
+		rec, err := csvReader.Read()
+		if err == io.EOF {
```

Comment:
Also this looks very similar to the above file too, could we combine these as well?